starter-code/src/components/BeersDetails.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import axios from "axios";
+import Nav from "./Nav";
+import DetailsCard from "./DetailsCard";
+
+class BeersDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      beers: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get(
+        `https://ih-beers-api2.herokuapp.com/beers/${this.props.match.params._id}`
+      )
+      .then(response => {
+        this.setState({
+          beers: response.data
+        });
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        <Nav />
+        <DetailsCard
+          name={this.state.beers.name}
+          image={this.state.beers.image_url}
+          tagline={this.state.beers.tagline}
+          brewed={this.state.beers.first_brewed}
+          level={this.state.beers.attenuation_level}
+          contributed={this.state.beers.contributed_by}
+          description={this.state.beers.description}
```

Comment:
as you see, you are repeating this.state.beers every line, i would send this.state.beers as props so it reduce this line to one line like this
beers={this.state.beers}